database/migrations/2021_06_14_055446_create_mymontys_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateMymontysTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('mymontys', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('cookie_id', 100);
+            $table->string('monty', 100);
+            $table->string('outcome', 100);
+            $table->string('softdeletes', 100);
```

Comment:
Lookup Laravel softdeletes, this is not created correctly. It is $table->softDeletes(); after you do Use SoftDeletes.